sched/scheduler/stateful_scheduler.go

```diff
@@ -145,6 +146,7 @@ func NewStatefulScheduler(
 		runnerFactory: rf,
 		asyncRunner:   async.NewRunner(),
 		addJobCh:      make(chan jobAddedMsg, 1),
+		killJobCh:     make(chan string, 1),
```

Comment:
Dumb question because I'm probably missing something: if addJobCh and killJobCh have a buffer size of 1, and they only get drained once per scheduler loop, what happens if we get more than one request per loop?